lib/templates-sync.php

```diff
@@ -166,7 +166,7 @@ function gutenberg_clear_synchronize_last_checks() {
  * @param int     $post_id ID of the deleted post.
  * @param WP_Post $post WP_Post instance of the deleted post.
  */
-function gutenberg_clear_synchronize_last_checks_after_delete( $postid, $post ) {
+function gutenberg_clear_synchronize_last_checks_after_delete( $post_id, $post ) {
```

Comment:
this was causing a phpcs issue, just making sure we have it here to test that phpunit is good